receiver/jmxreceiver/config.go

```diff
@@ -263,6 +265,19 @@ func (c *Config) Validate() error {
 		}
 	}
 
+	if c.PasswordFile != "" {
+		info, err := os.Stat(c.PasswordFile)
+		if err != nil {
+			return fmt.Errorf("`password_file` is inaccessible: %w", err)
+		}
+		switch info.Mode().Perm() {
+		// Matches JMX agent requirements for password file.
+		case 0400, 0600:
```

Comment:
can we make these perms macros like: 
``` OWNER_READ = 0400```, ```OWNER_RW = 0600```

Reply:
Yeah it makes sense for readability do this, but unfortunately isn't common practice.